gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectAttributes.java

```diff
@@ -101,7 +137,7 @@ public FileTime lastModifiedTime() {
    */
   @Override
   public ImmutableMap<String, String> userMetadata() {
-    return ImmutableMap.copyOf(metadata.getOptions().getUserMetadata());
+    return ImmutableMap.copyOf(info.metadata());
```

Comment:
As discussed in a previous comment, I don't think it's necessary for this class to be refactored to make `info` nullable.